<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>humhumClass</class>
 <widget class="QMainWindow" name="humhumClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>humhum</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background:#EEEEF2
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:#007ACC;
}</string>
    </property>
    <property name="text">
     <string>曲库</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:#007ACC;
}</string>
    </property>
    <property name="text">
     <string>识别结果</string>
    </property>
   </widget>
   <widget class="QListView" name="list_mids">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>300</width>
      <height>400</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QListView{
border: 2px solid grey;   
border-radius: 5px;   
background-color: #FFFFFF;
}</string>
    </property>
   </widget>
   <widget class="QListView" name="list_results">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListView{
border: 2px solid grey;   
border-radius: 5px;   
background-color: #FFFFFF;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_rec">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>440</y>
      <width>61</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background:#FF1847;
	border-radius:25px;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>REC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_hum">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>440</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background:#007ACC;
	border-radius:10px;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>Hum 识别</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="humhum.qrc"/>
 </resources>
 <connections/>
</ui>
